control/control.MicroOS.xml

```diff
@@ -71,7 +72,7 @@ textdomain="control"
         <rle_offer_rulevel_4 config:type="boolean">true</rle_offer_rulevel_4>
 
         <!-- FATE: #303893: Default to enabled kdump -->
```

Comment:
Comment no longer matches

Reply:
I have not been sure if "default" means that without defining this entry the default has been true.